### PR TITLE
A new draft for the AI settings page (DOC-7867)

### PR DESCRIPTION
**Pull request by paperclypse**
This is for the grace period and inactive issue settings.

<sup> Created from VS Code using [CodeStream](https://codestream.com/?utm_source=cs&utm_medium=pr&utm_campaign=github*com)</sup>

### CODE REVIEW

File: src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/user-settings.mdx [hunk 1]
```diff
@@ -0,0 +1,46 @@
+---
+title: "Incident intelligence user settings."
+metaDescription: "Use the incident intelligence settings to set your grace period and inactive issue time policies."
+---
+
+Incident intelligence general settings page lets you set how much time our system spends observing new activity for the purposes of correlation. It also lets you decide how much time should pass before inactive issues are closed.
```
Comment (barbnewrelic) [inline src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/user-settings.mdx:6]: "The incident intelligence general settings page"? Should any of this be bold or initial cap per style guide?

Comment (barbnewrelic) [reply src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/user-settings.mdx:6]: Also "purposes of correlation" I would think there would be more than one purpose besides correlation? Or just "purpose"?

File: src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/user-settings.mdx [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+To find your user settings, go to **one.newrelic.com > Alerts & AI > General**.
```
Comment (barbnewrelic) [inline src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/user-settings.mdx:8]: style guide says the pathing should include the url link

File: src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/user-settings.mdx [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+The grace period is the time between when an issue is created and when it's activated. For each priority level, you can set the grace period between 0 and 20 minutes. 
+
+If you've set your incident preference to `By condition and signal`, your grace period is always 0 minutes.
+
+The higher the time period, the more effectively our system analyzes new activity for correlation and to eliminate noise.
+
+Notifications are sent when the grace period time has elapsed.
+
+These settings apply to all data coming in to your account.
+
```
Comment (barbnewrelic) [inline src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/user-settings.mdx:21]: Maybe it's just me, but it seems odd that each of your sentences is a paragraph.

File: src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/user-settings.mdx [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+The default values are:
+
+* Low: 15 minutes
+* Medium: 10 minutes
+* High: 5 minutes
+* Critical: 1 minute
+
+## Inactive issues policy [#inactive-issues]
+
+The inactive issues policy controls how much time elapses before an  inactive issue is closed. The range is from 1-30 days and the default value is 3 days.
```
Comment (barbnewrelic) [inline src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/user-settings.mdx:31]: extra space between "an" and "inactive"

**Review by barbnewrelic — APPROVED** [review]
Style guide comments, otherwise looks fine. I don't need to review this again after you decide what to do with my review comments. Go ahead and merge when you're ready! Thanks for making this simple!

File: src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/user-settings.mdx (hunk 1 above)
Comment (paperclypse) [reply src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/user-settings.mdx:6]: I don't think it should be bolded. Also, we don't initial cap "incident intelligence" anymore.

Comment (barbnewrelic) [reply src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/user-settings.mdx]: oh, i thought "settings" was the actual name of the UI page. Never mind!

**Pull request merged**